.github/workflows/ci.yml

```
@@ -13,7 +13,21 @@ jobs:
     strategy:
       matrix:
         go-version: [1.15.x, 1.16.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest]
+
+    services:
+      mysql:
+        image: postgres
```

probably

```suggestion
      postgres:
        image: postgres
```

:grimacing: